local/bin/js/algolia-index-sync.js

```diff
@@ -10,7 +10,7 @@ const getIndexName = () => {
             algoliaIndexName = configDocs['live'].algoliaConfig.index;
             break;
         case 'preview':
-            algoliaIndexName = configDocs['preview'].algoliaConfig.index;
+            algoliaIndexName = 'docs_preview_nick';
```

Comment:
```suggestion
            algoliaIndexName = configDocs['preview'].algoliaConfig.index;
```
apply this before merging